samples/Cryptocash.Api/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Infrastructure.Persistence/CryptocashDbContext.g.cs

```diff
@@ -116,6 +117,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             if (type != null)
             {
                 ((INoxDatabaseConfigurator)_dbProvider).ConfigureEntity(codeGeneratorState, new EntityBuilderAdapter(modelBuilder.Entity(type)), entity);
+
+                if (entity.Keys.Count == 1 &&
+                    entity.GetAttributesToLocalize().Any())
```

Comment:
I would remove this if completely, because  keys.Count is already validated on the solution and GetAttributesToLocalize is used only for count, I guess inside ConfigureLocalizedEntity you are doing the same, so iterating the collection is enough...

Reply:
Done